kf_lib_data_ingest/app/cli.py

```diff
@@ -7,10 +7,7 @@
 
 import click
 
-from kf_lib_data_ingest.config import (
-    DEFAULT_TARGET_URL,
-    DEFAULT_LOG_LEVEL
-)
+from kf_lib_data_ingest.config import DEFAULT_LOG_LEVEL, DEFAULT_TARGET_URL
```

Comment:
I thought we agreed on isort multiline output mode of 3 - Vertical Hanging Indent?

Reply:
isort mode "...defines how from imports wrap **when they extend past the line_length limit**"

Reply:
Oh I see..I think we'll need to update the rest of the code base then at some point. I've been doing vertical hanging indent whenever I have more than 2 imports from a package

Reply:
lol

Reply:
I just like the way it looks 🤷‍♀️